__tests__/app.test.js

```diff
@@ -26,4 +23,18 @@ describe('/api/topics', () => {
             expect(topic).toHaveProperty("description", expect.any(String))
         })
     })
+})
+
+describe('/api', () => {
+    test('should return an object with all the available endpoints of the api', async () => {
+        const { body: { endpoints } } = await request(app)
+            .get('/api')
+            .expect(200)
+        expect(endpoints).toBeInstanceOf(Object)
+        expect(endpoints).toHaveProperty("GET /api")
+        expect(endpoints).toHaveProperty("GET /api/topics")
```

Comment:
Whilst these ```.toHaveProperty``` tests won't break one you add more endpoints to the json, I would still leave them out unless you want to keep updating new endpoints on the test. Taking them out will save you some work 🙂